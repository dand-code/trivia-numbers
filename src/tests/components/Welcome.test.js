import React from 'react';
import { create } from 'react-test-renderer';

function Welcome(props) {
    return `<WelcomeWrapper>
  <SubTitle>
      Welcome to Trividado number quiz!
  </SubTitle>
  <Link to="/trivial">
      <Button
          target="_blank"
          rel="noopener noreferrer"
      >
          Start
    </Button>
  </Link>
</WelcomeWrapper >`;
}

describe("Welcome component", () => {
    test("Matches the snapshot", () => {
        const welcome = create(<Welcome />);
        expect(welcome.toJSON()).toMatchSnapshot();
    });
});