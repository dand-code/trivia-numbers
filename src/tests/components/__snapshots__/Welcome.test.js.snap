// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component Matches the snapshot 1`] = `
"<WelcomeWrapper>
  <SubTitle>
      Welcome to Trividado number quiz!
  </SubTitle>
  <Link to=\\"/trivial\\">
      <Button
          target=\\"_blank\\"
          rel=\\"noopener noreferrer\\"
      >
          Start
    </Button>
  </Link>
</WelcomeWrapper >"
`;
